docs/apis/calcite.md

```diff
@@ -0,0 +1,529 @@
+
+              }
+
+            }
+
+          }
+
+        ]
+
+      }
+
+**model_file_csv.json**
+
+This example will read from input data from location in HDFS mentioned as "directory" in model_file_csv.json. It can be either directory / file . To change input directory you need to relaunch an application by modifying model_file_csv.json. Schema format is provided as input through property “schema” and schema name is mentioned as property “name”.
+
+    @Override
+
+    public void populateDAG(DAG dag, Configuration conf)
+
+    {
+
+     String modelFile = conf.get("modelFile");
+
+     String model;
+
+     try {
+
+       model = FileUtils.readFileToString(new File(modelFile));
+
+     } catch (IOException e) {
+
+       throw new RuntimeException(e);
+
+     }
+
+     SQLExecEnvironment.getEnvironment()
+
+         .withModel(model)
+
+         .executeSQL(dag, conf.get("sql"));
+
+    }
+
+In above code snipped we pass model file configuration to SQLExecEnvironment and dag and sql to be executed as input. 
+
+![image alt text](images/image_1.jpg)
+
+**Physical Dag View of Application.**
+
+## SQL Application with API:
+
+In previous example we used model file to configure SQLExecution environment . But in this example we will configure environment by configuring input table explicitly through code.
+
+    <configuration>
+
+     <!-- Input Definition -->
+
+     <property>
+
+       <name>csvSchemaInName</name>
+
+       <value>ORDERS</value>
+
+     </property>
+
+     <property>
+
+       <name>csvSchemaIn</name>
+
+       <value>{"separator":",","quoteChar":"\"","fields":[{"name":"RowTime","type":"Date","constraints":{"format":"dd/MM/yyyy hh:mm:ss Z"}},{"name":"id","type":"Integer"},{"name":"Product","type":"String"},{"name":"units","type":"Integer"}]}</value>
+
+     </property>
+
+     <property>
+
+       <name>sourceFile</name>
+
+       <value>/user/hadoop/inputs/input.csv</value>
+
+     </property>
+
+     <!-- Execution SQL -->
+
+     <property>
+
+       <name>sql</name>
+
+       <value>SELECT STREAM ROWTIME, PRODUCT FROM ORDERS</value>
+
+     </property>
+
+    </configuration>
+
+**Configuration file for an application**
+
+ In above configuration file we have explicitly declared properties like csvSchemaInName, csvSchemaIn,sourceFile (input file from HDFS) and sql to be executed .
+
+     @Override
+
+     public void populateDAG(DAG dag, Configuration conf)
+
+     {
+
+       // Source definition
+
+       String schemaInName = conf.get("csvSchemaInName");
+
+       String schemaIn = conf.get("csvSchemaIn");
+
+       String sourceFile = conf.get("sourceFile");
+
+       SQLExecEnvironment.getEnvironment()
+
+           .registerTable(schemaInName, new FileEndpoint(sourceFile, new CSVMessageFormat(schemaIn)))
+
+           .executeSQL(dag, conf.get("sql"));
+
+     }
+
+![image alt text](images/image_1.jpg)
+
+
+** Physical Dag View of Application **
```

Comment:
Where is the Physical DAG view of application?
If not there.. Remove the section...